src/payment/payment-methods/cybersource/service.js

```diff
@@ -64,6 +61,84 @@ function getCardNumberDigits(cardNumber) {
   return cardNumberDigitArray.join('');
 }
 
+function getPaymentToken(microformOptions) {
+  return new Promise((resolve, reject) => {
+    if (!window.microform) {
+      reject(Error('Microform not initialized'));
+    }
+    window.microform.createToken(microformOptions, (err, token) => {
+      if (err) {
+        reject(err);
```

Comment:
This includes card number validation errors, we need to handle those explicitly and display a validation error message to the user.

https://developer.cybersource.com/api/developer-guides/dita-flex/SAFlexibleToken/FlexMicroform/api_reference/class_MicroformError.html